packages/react-ui-components/README.md

```diff
@@ -0,0 +1,16 @@
+# stencil-ds-react-template
```

Comment:
This should be updated with useful information like: How to use, ...
Please add a TODO comment.  So that we don't forget